tools/ivar/consensus.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="ivar-consensus" name="iVar-Consensus" version="@VERSION@">
+    <description> In Progress!</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code">
+<![CDATA[
+
+#set $outname = '{}.consensus'.format(os.path.splitext($input_bam.name)[0])
+
+samtools mpileup -A -d 6000000 -B -Q 0 --reference '$input_reference' '$input_bam' | ivar consensus -p '$outname'
```

Comment:
The `-d 6000000` is the max depth option which should be specified as an advanced option in the wrapper instead of a hard coded value.

Reply:
Also ivar itself defaults to 0 which is 8000